src/card.css.ts

```diff
@@ -395,7 +395,3 @@ export const flexColumn = style({
     display: 'flex',
     flexDirection: 'column',
 });
-
-globalStyle(`${flexColumn} > :last-child`, {
```

Comment:
the problem with this rule is that `:last-child` uses dom order, not display order, so it was applying the styles to headline because was the last one in dom order